src/Maestro/Maestro.Data/Maestro.Data.csproj

```diff
@@ -19,8 +19,10 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Microsoft.DotNet.Darc\src\DarcLib.AzDev\Microsoft.DotNet.DarcLib.AzDev.csproj" />
+    <ProjectReference Include="..\..\Microsoft.DotNet.ServiceFabric.ServiceHost\Microsoft.DotNet.ServiceFabric.ServiceHost.csproj" />
```

Comment:
The "data" library shouldn't be taking dependencies on the AzureDevOps and ServiceFabric libraries.  It seems like we are getting a tangled web of dependencies there that probably needs to be thought about a bit.

Reply:
Sounds like I might need a new project then? (Between this and the targeting issue.) I'm flexible about where the logic ends up, but it can't stay in SubscriptionActorService (and I'm not sure it should since it's the implementation for any project on the server that needs to do Darc things).